### PR TITLE
Fix actionlint workflow

### PR DESCRIPTION
**Pull request by szepeviktor**
[Currently](https://github.com/chainguard-dev/actions/commit/466fa2b2f99fb5d1a1dc89500590ecb9273efc80) actionlint's `find` command does not output the list of YAML files.

🥊 There is a huge problem. `actionlint` works on workflow files only.
```
"on" section is missing in workflow
```
`action.yaml` files in this repository are actions, not workflows.
https://github.com/rhysd/actionlint/issues/46

### CODE REVIEW

**Comment by szepeviktor** [comment]
So currently we could lint `.github/workflows/*` files only.

File: .github/workflows/actionlint.yaml [hunk 1]
```diff
@@ -19,8 +19,7 @@ jobs:
         uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
       - id: get_yamls
         run: |
-          yamls=$(find . -name "*.yaml" -iname '.chainguard/source.yaml')
-          echo "files="${yamls}"" >> $GITHUB_OUTPUT
+          echo "files=$(find ./ -type f -name '*.yaml' -not -path './.chainguard/source.yaml')" >> "${GITHUB_OUTPUT}"
```
Comment (cpanato) [inline .github/workflows/actionlint.yaml:22]:
```suggestion
          echo "files=$(find ./ -type f -name '*.y*ml' -not -path './.chainguard/source.yaml')" >> "${GITHUB_OUTPUT}"
```

maybe?

**Review by cpanato — CHANGES_REQUESTED** [review]
check my comment and also please rebase

**Comment by szepeviktor** [comment]
Done 🍏

**Comment by k4leung4** [comment]
obsolete with https://github.com/chainguard-dev/actions/pull/458

**Pull request closed without merging**